src/main/java/seedu/address/model/country/CountryManager.java

```diff
@@ -54,10 +60,11 @@ public static boolean isValidCountryCode(String countryCode) {
      * @return Whether {@code country} contains {@code countryNote}.
      */
     public boolean hasCountryNote(Country country, Note countryNote) {
-        if (!isValidCountryCode(country.getCountryCode())) {
+        if (!countryNotesMap.containsKey(country)) {
             return false;
         }
```

Comment:
This should be an assert / exception instead, since our countryNotesMap already contains all possible countries, and if there is a country not contained in our map, it should not have been created in the first place